changelogs/unreleased/gh-6144-incorrect-timeout-error.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/net.box
+
+* Fixed bug with the type of the error that occurs when timeout expires in net.box functions  (gh-6144).
```

Comment:
Two spaces between 'functions' and '(gh-6144)'

Reply:
Fixed